client/log_verifier.go

```diff
@@ -56,13 +59,19 @@ func NewLogVerifierFromTree(config *trillian.Tree) (*LogVerifier, error) {
 		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): NewLogHasher(): %v", err)
 	}
 
-	// Log Key
+	// Log Pub Key
 	logPubKey, err := der.UnmarshalPublicKey(config.GetPublicKey().GetDer())
 	if err != nil {
 		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): Failed parsing Log public key: %v", err)
 	}
 
-	return NewLogVerifier(logHasher, logPubKey), nil
+	// Log Sig Hash
```

Comment:
As above; also, it would be nice if these comments said something that the code doesn't (e.g. "Extract the hash algorithm (which will be used on the inputs for signature generation) from the tree configuration.")

Reply:
Removed for redundancy.  
Assuming `SigHash` is enough documentation?